tests/api/test_static.py

```diff
@@ -1,5 +1,5 @@
 def test_static_css(dashboard):
-    response = dashboard.get('dashboard/static/css/app.css')
+    response = dashboard.get('dashboard/static/css/main.css')
```

Comment:
why this rename?

Reply:
One of our package updates that outputs our css file has changed to output "main.css" instead of "app.css" - so we refer to the new one now